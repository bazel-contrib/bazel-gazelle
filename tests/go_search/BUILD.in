# gazelle:prefix example.com/use
# gazelle:go_search third_party
# gazelle:go_search replace/b example.com/b
