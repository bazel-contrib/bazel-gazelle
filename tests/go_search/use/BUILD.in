# gazelle:go_search ../third_party
