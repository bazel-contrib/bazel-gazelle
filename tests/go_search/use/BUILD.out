load("@io_bazel_rules_go//go:def.bzl", "go_library")

# gazelle:go_search ../third_party

go_library(
    name = "use",
    srcs = ["use.go"],
    importpath = "example.com/use/use",
    visibility = ["//visibility:public"],
    deps = [
        "//replace/b/p",
        "//third_party/example.com/a/p",
    ],
)
